### PR TITLE
Schema without product

### PR DESCRIPTION
**Pull request by fmhf**
Update schema without product.

### CODE REVIEW

File: catalog.uml [hunk 1]
```diff
@@ -110,74 +116,199 @@ entity "store_geofencing" as e_geofencing {
...
-  *sales_channel_id_id: id <<FK>>
-  condition: <<enumeration>>[always,recommended,never]
-}
+  difficulty_level: number(uint)
+  complexity_scale: number(uint)
+  spiciness_level: number(uint)
 
+  suitable_for_season: []text
+
+  primary_protein: text
```
Comment (frostcs7) [inline catalog.uml:218]: Do we need to consider these as enums as well?

File: catalog.uml [hunk 2]
```diff
@@ -110,74 +116,199 @@ entity "store_geofencing" as e_geofencing {
...
+  dish_type: dish_type(text)
+  'concepts
+  behavioral_concept: [] behavioral_concept(text)
+  dietary_concept: []dietary_concept(text)
+  marketing_concept: []marketing_concept(text)
+
+  'backwards compatibility to Recipe's CPS
+  cooking_method: []text
+  secondary_protein: text
+  secondary_starch: text
```
Comment (frostcs7) [inline catalog.uml:264]: Same here: Do we need to consider these as enums as well?

**Review by frostcs7 — COMMENTED** [review]
commented

File: catalog.uml (hunk 1 above)
Comment (fmhf) [reply catalog.uml:218]: You're right. In CPS we use predefined values for the recipe content.

File: catalog.uml (hunk 2 above)
Comment (frostcs7) [reply catalog.uml:264]: Lets change this all the text to enums where needed :)

**Review by frostcs7 — APPROVED** [review]

**Pull request merged**